XamlControlsGallery/ControlPages/IconControlsPage.xaml

```diff
@@ -0,0 +1,116 @@
+            <local:ControlExample.Example>
+                <StackPanel>
+                <TextBlock TextWrapping="WrapWholeWords">
+                       The ShowAsMonochrome property (true by default) will result in a solid block of the foreground color if the property is set to true and the icon is more than one color. This behavior can be ignored by setting the ShowAsMonochrome property to false.<LineBreak></LineBreak>
+                </TextBlock>
+                    <BitmapIcon x:Name="SlicesIcon" UriSource="ms-appx:///Assets/slices.png" Width="50" HorizontalAlignment="Left" ShowAsMonochrome="False"/>
+                </StackPanel>
+            </local:ControlExample.Example>
+            <local:ControlExample.Options>
+                <CheckBox x:Name="MonochromeButton" Content="Monochrome" IsChecked="{x:Bind SlicesIcon.ShowAsMonochrome, Mode=TwoWay}"/>
+            </local:ControlExample.Options>
+            <local:ControlExample.Xaml>
+                <x:String>
+                    &lt;BitmapIcon x:Name="SlicesIcon" UriSource="ms-appx:///Assets/slices.png" Width="50" ShowAsMonochrome="$(ShowAsMonochrome)"/&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+            <local:ControlExample.Substitutions>
+                <local:ControlExampleSubstitution Key="ShowAsMonochrome" Value="{ x:Bind MonochromeButton.IsChecked.Value, Mode=OneWay}"/>
+            </local:ControlExample.Substitutions>
+        </local:ControlExample>
+        <local:ControlExample x:Name="Example3" HeaderText="A FontIcon using a glyph from a specific font family in a button" XamlSource="Icons/FontIconSample1_xaml.txt">
+            <local:ControlExample.Example>
+                <StackPanel>
+                    <TextBlock TextWrapping="WrapWholeWords">Use FontIcon as the icon for a control if you want to specify a Glyph value from a FontFamily. Windows 10 uses the Segoe MDL2 Assets FontFamily.<LineBreak></LineBreak></TextBlock>
+                    <Button x:Name="ExampleButton1">
+                    <StackPanel>
+                        <FontIcon FontFamily="Segoe MDL2 Assets" Glyph="&#xE790;"/>
+                        <TextBlock Text="Select a color"/>
+                    </StackPanel>
+                </Button>
+                </StackPanel>
+            </local:ControlExample.Example>
+        </local:ControlExample>
+        <local:ControlExample x:Name="Example4" HeaderText="A ImageIcon using a bitmap image in a button" XamlSource="Icons/ImageIconSample1_xaml.txt">
+            <local:ControlExample.Example>
+                <StackPanel>
+                    <TextBlock TextWrapping="WrapWholeWords">To use an ImageIcon as the icon for a control, you can set image that has a file format supported by the Image class. The two examples here show a PNG and SVG image as the icon.<LineBreak></LineBreak></TextBlock>
+                    <Button x:Name="ImageExample1" Width="100">
```

Comment:
I was avoiding using an AppBarButton on this page only because AppBarButton has its own examples so I didn't want to mix and match where AppBarButton was used.